Example_App/www/js/index.js

```diff
@@ -21,97 +21,233 @@
 }
-document.getElementById("register-user-button").addEventListener("click", registerUser);
+document
+  .getElementById("register-user-button")
+  .addEventListener("click", registerUser);
 
 function registerUser() {
-    Optimove.registerUser(document.getElementById("text-area-user-id").value, document.getElementById("text-area-user-email").value).then(() => {
-        document.getElementById("text-area-user-id").value = "";
-        document.getElementById("text-area-user-email").value = "";
-        success();
-    }, error);
-
+  Optimove.registerUser(
+    document.getElementById("text-area-user-id").value,
+    document.getElementById("text-area-user-email").value
+  ).then(() => {
+    document.getElementById("text-area-user-id").value = "";
+    document.getElementById("text-area-user-email").value = "";
+    success();
+  }, error);
 }
-document.getElementById("get-visitor-id-button").addEventListener("click", getVisitorId);
+document
+  .getElementById("get-visitor-id-button")
+  .addEventListener("click", getVisitorId);
 
 function getVisitorId() {
-    Optimove.getVisitorId().then((visitorId) => {
+  Optimove.getVisitorId().then((visitorId) => {
+    document.getElementById("text-area-visitor-id").value = visitorId;
+  }, error);
+}
+
+
+
 
-        document.getElementById("text-area-visitor-id").value = visitorId;
+document
+  .getElementById("push-register-button")
+  .addEventListener("click", pushRegister);
 
-    }, error);
+function pushRegister() {
+  Optimove.pushRegister().then(success, error);
 }
-document.getElementById("get-current-user-identifier-button").addEventListener("click", getCurrentUserIdentifier);
+document
+  .getElementById("in-app-consent-true")
+  .addEventListener("click", inAppUpdateConsentTrue);
 
-function getCurrentUserIdentifier() {
-    Optimove.getCurrentUserIdentifier().then((visitorId) => {
-        document.getElementById("text-area-current-user-identifier").value = visitorId
-    }, error);
+function inAppUpdateConsentTrue() {
+  Optimove.inAppUpdateConsent(true).then(success, error);
 }
 
+document
+  .getElementById("in-app-consent-false")
+  .addEventListener("click", inAppUpdateConsentFalse);
+
+function inAppUpdateConsentFalse() {
+  Optimove.inAppUpdateConsent(false).then(success, error);
+}
+
+document
+  .getElementById("in-app-get-inbox-items-button")
+  .addEventListener("click", inAppGetInboxItems);
+
+function inAppGetInboxItems() {
+  Optimove.inAppGetInboxItems().then((inboxItemsArray) => {
+    items = inboxItemsArray;
+    alert(items);
+  }, error);
+}
+
+document
+  .getElementById("in-app-mark-all-inbox-items-as-read-button")
+  .addEventListener("click", inAppMarkAllInboxItemsAsRead);
+
+function inAppMarkAllInboxItemsAsRead() {
+  Optimove.inAppMarkAllInboxItemsAsRead().then(success, error);
+}
+
+document
+  .getElementById("in-app-mark-inbox-item-as-read-button")
+  .addEventListener("click", inAppMarkAsRead);
+
+function inAppMarkAsRead() {
+  Optimove.inAppMarkAsRead(
```

Comment:
this accepts item, not an id. Worth updating ExampleApp?

Reply:
sure, done